src/main/kotlin/org/kamiblue/client/module/modules/chat/AntiSpam.kt

```diff
@@ -21,14 +21,14 @@ internal object AntiSpam : Module(
     private val page = setting("Page", Page.TYPE)
 
     /* Page One */
-    private val discordLinks = setting("Discord", true, { page.value == Page.TYPE })
-    private val slurs = setting("Slurs", true, { page.value == Page.TYPE })
-    private val swears = setting("Swears", false, { page.value == Page.TYPE })
-    private val automated = setting("Automated", true, { page.value == Page.TYPE })
-    private val ips = setting("Server Ips", true, { page.value == Page.TYPE })
-    private val specialCharEnding = setting("Special Ending", true, { page.value == Page.TYPE })
-    private val specialCharBegin = setting("Special Begin", true, { page.value == Page.TYPE })
-    private val greenText = setting("Green Text", false, { page.value == Page.TYPE })
+    private val discordLinks = setting("Discord", true, { page.value == Page.TYPE }, description = "Block messages containing discord links")
+    private val slurs = setting("Slurs", true, { page.value == Page.TYPE }, description = "Removes slurs from messages")
+    private val swears = setting("Swears", false, { page.value == Page.TYPE }, description = "Removes swear words from messages")
```

Comment:
I feel these specifically are self explanatory and don't warrant a description, and adding unnecessary descriptions increases the workload for future translations